Events/Ch25.event

```diff
@@ -65,6 +75,9 @@ END_MAIN
 
 MiscBasedEvents:
 CauseGameOverIfLordDies
+DefeatBoss(OhNoMikeIsDead)
```

Comment:
Change the DeathQuoteEditor table to have an entry for the boss on this chapter. (You can make the textID placeholder t ext if need be)